### PR TITLE
scala 2.12.10 and fixing oracle build

### PR DESCRIPTION
**Pull request by francisdb**

### CODE REVIEW

**Comment by francisdb** [comment]
@hseeberger can you check if this fixes the build on dockerhub?

**Comment by francisdb** [comment]
ping 🔔

File: oracle/Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 # Pull base image
 ARG BASE_IMAGE_TAG
-FROM openjdk:${BASE_IMAGE_TAG:-8u212-b04-jdk-stretch}
+FROM openjdk:${BASE_IMAGE_TAG:-11.0.2-jdk-oraclelinux7}
```
Comment (hseeberger) [inline oracle/Dockerfile]: Why default to 11 here and 8 above (debian)?

File: oracle/Dockerfile [hunk 2]
```diff
@@ -19,8 +19,8 @@ RUN \
   curl -L -o sbt-$SBT_VERSION.rpm https://bintray.com/sbt/rpm/download_file?file_path=sbt-$SBT_VERSION.rpm && \
   rpm -i sbt-$SBT_VERSION.rpm && \
   rm sbt-$SBT_VERSION.rpm && \
-  yum update && \
-  yum install sbt
+  yum -y update && \
```
Comment (hseeberger) [inline oracle/Dockerfile]: Nice! Did that ever work before?

Comment (francisdb) [reply oracle/Dockerfile]: nope, must not have been tested as it prompted if it was ok to download

File: oracle/Dockerfile (hunk 1 above)
Comment (francisdb) [reply oracle/Dockerfile]: This file with it's defaults was totally broken, using rpm on debian

- In the vars.sh there was only `12.0.1-jdk-oraclelinux7` which did not work for me so I brought it back to the same version as in use on the debian images (LTS jdk11)
- The lowest available openjdk jdk-oraclelinux7 seems to be 10.0.2 so 8 is not an option)
- There were several issues with newer versions so I settled for this one.

**Comment by hseeberger** [comment]
I'll give it a try, although I do not like that we have two versions of Java 11 (stretch and oraclelinux7).

**Comment by francisdb** [comment]
I'm also not a fan of this Oracle stuff, but you accepted that pr?

**Comment by hseeberger** [comment]
Well, I do not have enough time for this project and hence have happily accepted a lot of PRs ;-)

Looks like going forward there's Oracle only: https://github.com/docker-library/docs/blob/master/openjdk/README.md#supported-tags-and-respective-dockerfile-links

**Comment by hseeberger** [comment]
Build worked, BTW.

**Comment by francisdb** [comment]
> Well, I do not have enough time for this project and hence have happily accepted a lot of PRs ;-)
> 
> Looks like going forward there's Oracle only: https://github.com/docker-library/docs/blob/master/openjdk/README.md#supported-tags-and-respective-dockerfile-links

They still have debian (buster) builds?

**Comment by francisdb** [comment]
There's also these

https://hub.docker.com/_/adoptopenjdk

**Comment by francisdb** [comment]
Do you want to provide non-lts jdk images, like the just released jdk 13?

**Comment by hseeberger** [comment]
I do not think so.

**Pull request merged**